test/y2storage/guided_proposal_test.rb

```diff
@@ -161,7 +161,8 @@
 
       it "creates the needed partitions with correct device names" do
         proposal.propose
-        raid = proposal.devices.dm_raids.first
+        # note: potentially order dependent; thera are two raids defined
```

Comment:
`Devicegraph#dm_raids` should return the list sorted by device name. So it shouldn't be any order dependency here.

Reply:
Well, according to the log it tries the second raid first, for whatever reason. This is just a note for the next developer.